stock_picking_batch_actual_date/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Picking Patch Actual Date",
+    "version": "16.0.1.0.0",
+    "author": "Quartile, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "category": "Stock",
+    "license": "AGPL-3",
+    "depends": ["stock_move_actual_date", "stock_picking_batch"],
```

Comment:
Odoo standard module should come first.
```suggestion
    "depends": ["stock_picking_batch", "stock_move_actual_date"],
```